### ISSUE TITLE
Error starting Julia due to old OpenBLAS library

### ISSUE THREAD

**Issue by GlenHertz** [JuliaLang/julia#34097]
Issue #29652 was closed due to the issue being with upstream OpenBLAS library [here](https://github.com/xianyi/OpenBLAS/issues/1947).  That issue has been fixed in OpenBLAS 0.3.6 but Julia would need to apply the patch or update the OpenBLAS library.  Please update to the latest OpenBLAS if possible to resolve this issue.

**Comment by GregPlowman** [comment]
I'm wondering if issue #33188 is related?

Julia v1.2.0 (and v1.3.0) doesn't start, but exits without error message. 
Replacing `libopenblas64_.dll` with the dll from Julia v1.1.1 appears to fix the problem and allows Julia to start.

The architecture of the problematic server is reported as `Skylake`, and some `AVX512` cpu features are reported as available and some are not (see below).

Could this OpenBLAS issue be the same problem here?
Is there anything I can do to further diagnose?
 
Running this code produced the following output:
```
using CpuId
cpuinfo()

avx512keys = (key for key in keys(CpuId.CpuFeatureDescription) if occursin("AVX512", string(key)))
for key in avx512keys
    println(key, " => ", cpufeature(key))
end
```

```
Cpu Property       Value
–––––––––––––––––– ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
Brand              Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz
Vendor             :Intel
Architecture       :Skylake
Model              Family: 0x06, Model: 0x55, Stepping: 0x04, Type: 0x00
Cores              16 physical cores, 32 logical cores (on executing CPU)
                   Hyperthreading detected
Clock Frequencies  2100 / 3700 MHz (base/max), 100 MHz bus
Data Cache         Level 1:3 : (32, 1024, 22528) kbytes
                   64 byte cache line size
Address Size       48 bits virtual, 46 bits physical
SIMD               512 bit = 64 byte max. SIMD vector size
Time Stamp Counter TSC is accessible via `rdtsc`
                   TSC runs at constant rate (invariant from clock frequency)
Perf. Monitoring   Performance Monitoring Counters (PMC) revision 4
                   Available hardware counters per logical core:
                   3 fixed-function counters of 48 bit width
                   4 general-purpose counters of 48 bit width
Hypervisor         No

AVX512FMAPS => false
AVX512VL => true
AVX512IFMA => false
AVX512CD => true
AVX512PF => false
AVX512ER => false
AVX512VBMI => false
AVX512F => true
AVX512VNNIW => false
AVX512BW => true
AVX512DQ => true
```

**Comment by GlenHertz** [comment]
I tried to use openblas from v1.1.1 and earlier but it didn't work.  I could get Julia to start by deleting `libopenblas*`.  I ran @GregPlowman's script and got the output:

```
  Cpu Property       Value                                                  
  –––––––––––––––––– –––––––––––––––––––––––––––––––––––––––––––––––––––––––
  Brand              Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz               
  Vendor             :Intel                                                 
  Architecture       :Skylake                                               
  Model              Family: 0x06, Model: 0x55, Stepping: 0x04, Type: 0x00  
  Cores              1 physical cores, 64 logical cores (on executing CPU)  
                     Hyperthreading detected                                
  Clock Frequencies  Not supported by CPU                                   
  Data Cache         Level 1:3 : (32, 1024, 28160) kbytes                   
                     64 byte cache line size                                
  Address Size       48 bits virtual, 46 bits physical                      
  SIMD               256 bit = 32 byte max. SIMD vector size                
  Time Stamp Counter TSC is accessible via `rdtsc`                          
                     TSC increased at every clock cycle (non-invariant TSC) 
  Perf. Monitoring   Performance Monitoring Counters (PMC) are not supported
  Hypervisor         Yes, Xen                                               

AVX512FMAPS => false
AVX512VL => false
AVX512IFMA => false
AVX512CD => false
AVX512PF => false
AVX512ER => false
AVX512VBMI => false
AVX512F => false
AVX512VNNIW => false
AVX512BW => false
AVX512DQ => false
```

If someone can let me know how to compile Julia with openblas v0.3.7 then I could give that a try.

**Comment by GlenHertz** [comment]
I was able to get things to work on Julia 1.3.0 by doing `]add OpenBAS_jll` and then copying over the OpenBLAS libraries into the `julia-1.3/lib/julia` folder (`cp ~/.julia/artifacts/95ada7f4942849227ce548f451c49c177fc7c860/lib/libopenblas64_.* lib/julia`).

**Comment by DilumAluthge** [comment]
Julia master/Julia nightly now uses a more recent version of OpenBLAS that fixes this bug, meaning that this will be fixed in Julia 1.5 and later.

Can this issue be closed? @ararslan @fredrikekre
